### DISCUSSION TITLE
Ollama function calling - FunctionChoiceBehavior.None()

### DISCUSSION THREAD

**Discussion by deepinderdeol** [microsoft/semantic-kernel#9851] [Q&A] [answered]
Even if I set FunctionChoiceBehavior.None() - functions still get executed. 
Is this a known issue?

**Comment by evchaki** [comment]
@RogerBarreto - can you take a look?

**Comment by rogerbarreto** [comment, marked as answer]
@deepinderdeol, if you are attempting to use this option against API's that don't support choice behaviors like Ollama, the SLM will not behave for other options that are not Auto (default behavior)

Let me know what is the connector you are using, this should work as expected for OpenAI/AzureOpenAI models.

**Reply by deepinderdeol** [reply]
Thanks @RogerBarreto - I was using the Ollama connector with models like llama3.2, qwen2 etc.
